appinfo/info.xml

```diff
@@ -2,9 +2,21 @@
 <info>
 	<id>files_mediaviewer</id>
 	<name>Mediaviewer</name>
-	<summary>Explore, show your image collections</summary>
-	<description>Mediaviewer adds image and video viewing capabilities</description>
-	<licence>GPL-2.0</licence>
+	<summary>Explore, show and share your media collections</summary>
+	<description>
+A media viewer that adds image and video viewing capabilities to Files view and Public Links.
+
+**Features**
+
+- Support for a large selection of image and video formats (depending on server setup)
+- Image and video viewer in one
```

Comment:
sounds a bit flat, would remove this line

Reply:
agree, done.